code/modules/client/preferences/species.dm

```diff
@@ -32,9 +32,14 @@
 	return values
 
 /datum/preference/choiced/species/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/prefs)
+	// SKYRAT EDIT START - Veteran-only races
+	var/datum/species/value_typepath = value
+	if(initial(value_typepath.veteran_only) && !is_veteran_player(target.client))
```

Comment:
```suggestion
	if(initial(value_typepath.veteran_only) && !is_veteran_player(target?.client))
```

Reply:
That's actually just checking if target exists